pkg/controller/plan/migration.go

```diff
@@ -1631,20 +1632,47 @@ func (r *Migration) updatePopulatorCopyProgress(vm *plan.VMStatus, step *plan.St
 		}
 
 		percent := float64(transferredBytes/0x100000) / float64(task.Progress.Total)
-		task.Progress.Completed = int64(percent * float64(task.Progress.Total))
+		newProgress := int64(percent * float64(task.Progress.Total))
+		err = r.checkPopulatorPod(newProgress, task.Progress.Completed, string(pvc.UID))
+		if err != nil {
+			return
```

Comment:
what kind of error do we expect here? I see we can get an error when the pod failed  (the one that is created in checkPopulatorPod) or an error that is returned by getPopulatorPods - should we return an error here when we fail to query for pods?

Reply:
depends on us... it does hints something is wrong. but if we will return nil in such case we will reconcile and try again. maybe better to log and reconcile.